common/tls/src/main/java/io/helidon/common/tls/ConfiguredTlsManager.java

```diff
@@ -291,6 +299,37 @@ protected TrustManagerFactory createTmf(TlsConfig tlsConfig) {
         }
     }
 
+    protected void initializeTmf(TrustManagerFactory tmf, KeyStore keyStore, TlsConfig tlsConfig)
```

Comment:
`protected` should have javadoc.

Reply:
This method should not throw checked exceptions (to be consistent with other methods in this class)